test/integration/cart_integration_test.rb

```diff
@@ -201,7 +213,50 @@ class CartIntegrationTest < ActionDispatch::IntegrationTest
     assert_equal project_path(project.id), current_path
   end
 
+  test "an authenticated user can add default amount to their cart" do
+    authenticated_user = create(:user)
+    ApplicationController.any_instance.stubs(:current_user).
+      returns(authenticated_user)
+    tenant = create(:tenant)
+    tenant.projects << create(:project)
+
+    visit "/#{tenant.organization}"
+    within(".row") do
+      click_link_or_button("Lend")
+    end
+
+    assert_equal '/pending_loan', current_path
+    within('#pending_loans') do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.